src/cart/crt_swim.c

```diff
@@ -55,7 +66,37 @@ CRT_RPC_DEFINE(crt_rpc_swim,  CRT_ISEQ_RPC_SWIM, /* empty */)
 CRT_RPC_DECLARE(crt_rpc_swim_wack, CRT_ISEQ_RPC_SWIM, CRT_OSEQ_RPC_SWIM)
 CRT_RPC_DEFINE(crt_rpc_swim_wack,  CRT_ISEQ_RPC_SWIM, CRT_OSEQ_RPC_SWIM)
 
-uint32_t crt_swim_rpc_timeout;
+uint32_t	 crt_swim_rpc_timeout;
+static bool	 crt_swim_should_fail;
+static uint64_t	 crt_swim_fail_delay;
+static uint64_t	 crt_swim_fail_hlc;
+static swim_id_t crt_swim_fail_id;
+
+static struct d_fault_attr_t *d_fa_swim_drop_rpc;
+
+static void
+crt_swim_fault_init(void)
+{
+	char *end;
+	char *env_val;
+
+	crt_swim_should_fail = false; /* disabled by default */
+	crt_swim_fail_hlc = 0;
+	crt_swim_fail_delay = 10;
+	crt_swim_fail_id = SWIM_ID_INVALID;
+
+	env_val = getenv("CRT_SWIM_FAIL_DELAY");
+	if (env_val) {
+		crt_swim_fail_delay = strtoul(env_val, &end, 0) ?: 10;
+		D_ERROR("*** CRT_SWIM_FAIL_DELAY=%lu\n", crt_swim_fail_delay);
+	}
+
+	env_val = getenv("CRT_SWIM_FAIL_ID");
```

Comment:
Isn't there an "attribute" field in the yaml file which is intended for parameters like this.

Reply:
Do you mean "argument"? Hmm. I can use it but two arguments will complicate parsing...

Reply:
That would be it.  You could always add a "argument2" to the field for the other one you need.